decidim-proposals/spec/system/admin/valuator_manages_proposals_spec.rb

```diff
@@ -113,5 +113,20 @@
       end
       expect(page).to have_content("successfully")
     end
+
+    it "have assign button" do
+      expect(page).to have_content("Assign")
+    end
+
+    it "stay in the same url after assignment evaluator" do
+      within "#js-form-assign-proposal-to-valuator" do
+        find("#valuator_role_id").click
+        find("option", text: another_user.name).click
+      end
+
+      click_button "Assign"
+
+      expect(current_url).to end_with(current_path)
+    end
```

Comment:
```suggestion
```

We should remove this test but add it to the `admin_manages_proposal_valuators_spec.rb` or `admin_manages_proposals_spec.rb`
The admin is the only user that should be allowed to add valuators